python/cuda_cccl/test/test_cuda_cccl.py

```diff
@@ -0,0 +1,63 @@
+import pytest
```

Comment:
Nice addition!

Is this run in the CI?

(For cuda_parallel and cuda_cooperative the tests are run from ci/test_python.sh)